src/Controller/Action/UpdateWishlistNameAction.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Controller\Action;
+
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\UpdateWishlistName;
+use BitBag\SyliusWishlistPlugin\Repository\WishlistRepositoryInterface;
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Component\Channel\Context\ChannelNotFoundException;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Messenger\Exception\HandlerFailedException;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class UpdateWishlistNameAction
+{
+    private MessageBusInterface $commandBus;
+
+    private FlashBagInterface $flashBag;
+
+    private TranslatorInterface $translator;
+
+    private ChannelContextInterface $channelContext;
+
+    private WishlistRepositoryInterface $wishlistRepository;
+
+    public function __construct(
+        MessageBusInterface $commandBus,
+        FlashBagInterface $flashBag,
+        TranslatorInterface $translator,
+        ChannelContextInterface $channelContext,
+        WishlistRepositoryInterface $wishlistRepository
+    ) {
+        $this->commandBus = $commandBus;
+        $this->flashBag = $flashBag;
+        $this->translator = $translator;
+        $this->channelContext = $channelContext;
+        $this->wishlistRepository = $wishlistRepository;
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $wishlistName = $request->request->get('edit_wishlist_name')['name'];
```

Comment:
there is a small hole (what if someone doesn't pass on the `name` ? an "Undefined offset: ..." exception will be thrown.

Reply:
Done. Added assertion.